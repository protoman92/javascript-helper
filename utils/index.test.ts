import { stripLeadingSlash } from ".";

describe("General utilities", () => {
  it("Strip leading slashes should work correctly", async () => {
    // Setup
    // When
    // Then
    expect(stripLeadingSlash("////abc")).toEqual("abc");
  });
});
