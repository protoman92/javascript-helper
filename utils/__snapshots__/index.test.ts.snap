// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`General utilities Deep clone should work correctly 1`] = `
Object {
  "a": Object {
    "b": Object {
      "c": Object {
        "d": "10",
        "e": "10",
      },
    },
  },
}
`;
